styleguide/source/_patterns/02-molecules/membership.twig

```diff
@@ -10,21 +10,16 @@
 %}
 
 {% if button %}
-<div class="ama__membership">
+    <div {{ setAttributes('ama__membership', attributes) }}>
```

Comment:
@nstriedinger is there a D8 piece to go with? Getting the following error when testing this in Drupal:
```
Twig_Error_Syntax: Unknown "setAttributes" function. in Twig_ExpressionParser->getFunctionNodeClass() (line 1 of /var/www/ama-d8.local/sg2-ama/twig/03-organisms/tools.twig).
```

Reply:
I did -- https://github.com/AmericanMedicalAssociation/ama-d8/pull/348 Pardon, I for to mention that @labbydev !